utils/riff/colors.js

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2018-present Riff Learning, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+/* eslint-disable header/header */
+
+/**
+ * List of color objects to use for nodes in the interaction network graph
+ * Each color object has a background and font color property in order to
+ * comply with accessbility requirements
+ */
+const networkGraphNodeColors = {
+    you: {
+        backgroundColor: '#8e7cc3', // purple
+        fontColor: '#fff',
+    },
+    course: {
+        backgroundColor: '#6aa84f', // green
+        fontColor: '#000',
+    },
+    learningGroupColors: [
+        {
+            backgroundColor: '#2b78e4', // blue
+            fontColor: '#000',
+        },
+        {
+            backgroundColor: '#f6b26b', // orange
+            fontColor: '#000',
+        },
+        {
+            backgroundColor: '#DB4C40', // red
+            fontColor: '#000',
+        },
+    ],
+};
+
+/**
+ * Append a hex opacity value to a hex color string
+ * Currently only supports 6 digit color string of the form
+ * '#rrggbb' e.g. '#4c338a'
+ *
+ * @example rgba(Colors.brightPlum, 0.5) returns '#ab45ab80'
+ *
+ * @param {string} hexColor
+ * @param {number} opacity - fractional opacity, from 0 (totally clear) to 1 (totally opaque)
+ *
+ * @returns {string} a hex color string including the opacity (#rrggbbaa)
+ *    e.g. '#ab45ab80'
+ */
+function rgbaColor(hexColor, opacity) {
+    // convert opacity to hex value
+    const hexOpacity = Math.round(0xff * opacity).toString(16);
+    return `${hexColor}${hexOpacity.length === 1 ? '0' : ''}${hexOpacity}`;
+}
+
+/**
+ * Get the color index to use for the nth other.
+ * Returns an index of an unknown array of colors
+ * Colors will be "reused" for values of n > the number of {colors}
```

Comment:
Should this be `for values of n > {colorsLength}`? There is no parameter named `colors`.